src/PuppeteerRunnerExtension.ts

```diff
@@ -15,8 +15,8 @@
  */
 
 import { Browser, ElementHandle, Frame, Page } from 'puppeteer';
-import { Frame as InternalFrame } from 'puppeteer/lib/esm/puppeteer/common/Frame.js';
-import { CDPPage as InternalPage } from 'puppeteer/lib/esm/puppeteer/common/Page.js';
+import { Frame as InternalFrame } from 'puppeteer-core/internal/common/Frame.js';
```

Comment:
does it work if puppeteer-core is not a peer dependency?

Reply:
Yea, because `puppeteer-core` is a dependency of `puppeteer` :) But we should probably list it as a dev dependency.

Reply:
I wonder if it works if puppeteer-core gets installed in node_modules/puppeteer/node_modules/puppeteer-core. I think the resolution might not work in this case.